mlf_core/create/templates/common_all_files/{{cookiecutter.common_all}}/.github/workflows/lint.yml

```diff
@@ -40,6 +40,17 @@ jobs:
         with:
           python-version: '3.9'
 
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          # This path is specific to Ubuntu
+          path: ~/.cache/pip
+          # Look to see if there is a cache hit for the corresponding requirements file
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
```

Comment:
Every time you use {{ }} you need to jinja2 escape with the raw statements in mentioned in the other PR